lsp-clients/n4js-vscode-extension/package.json

```diff
@@ -78,6 +78,18 @@
       {
         "command": "n4js.performance.collector.reset",
         "title": "N4JS: Reset collected performance data"
+      },
+      {
+        "command": "n4js.debug.printDebugInfo",
+        "title": "N4JS: Print Debug Information"
+      },
+      {
+        "command": "n4js.debug.setVerboseLevel2Error",
+        "title": "N4JS: Log Level: Error"
+      },
+      {
+        "command": "n4js.debug.setVerboseLevel2Info",
```

Comment:
In case this is just about log level, why not call the endpoint simply "n4js.debug.setLogLevel2Info"?